src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -2,12 +2,19 @@
 
 ## Unreleased
 
+* Fix an issue with getting sanitized category name in pass-through table name
+  mapping cases for `TldLogExporter`.
+  ([#1175](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1175))
+
 * TldLogExporter to export `SpanId` value in `ext_dt_spanId` field instead of
   `TraceId` value.
   ([#1184](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1184))
 * Add support for abstract domain sockets.
   ([#1199](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1199))
 
+* Update OTel SDK version to `1.5.0-alpha.1`.
```

Comment:
nit: Says alpha but it is actually doing `1.5.0-rc.1`?

Reply:
Fixed.